skiko/src/commonMain/kotlin/org/jetbrains/skiko/redrawer/Redrawer.kt

```diff
@@ -1,10 +1,15 @@
 package org.jetbrains.skiko.redrawer
 
+import kotlin.time.TimeSource
+
+private val initialTime = TimeSource.Monotonic.markNow()
+
 internal interface Redrawer {
     fun dispose()
-    fun needRedraw()
-    fun redrawImmediately()
+    fun needRedraw(throttledToVsync: Boolean)
+    fun redrawImmediately(updateNeeded: Boolean)
     fun syncBounds() = Unit
+    fun update(nanoTime: Long = initialTime.elapsedNow().inWholeNanoseconds)
```

Comment:
Might be `TimeSource.Monotonic.markNow().elapsedNow().inWholeNanoseconds` without extra global val